symfony.lock

```diff
@@ -228,6 +228,9 @@
     "phar-io/version": {
         "version": "2.0.1"
     },
+    "php": {
+        "version": "7.3"
```

Comment:
Is this supposed to be here?

Reply:
:man_shrugging: This was created by composer, I deleted it now.